packages/data-api/src/TupaiaDataApi.js

```diff
@@ -96,7 +96,7 @@ export class TupaiaDataApi {
     if (dataElementCodes && Array.isArray(dataElementCodes)) {
       const sqlQuery = await new SqlQuery(
         `
-        SELECT question.code, question.name, question.text
+        SELECT question.code, question.name, question.text, question.type
```

Comment:
Return question type for Custom Export to check if the data is binary and convert it from `1/0` to `Yes/No` for the values in the exported excel

Reply:
This creates a tighter coupling between our web-config-server client and the internal data-api. Our intention is that the data services would all be able to support the same response format, so that any client's code can be performed on results from any data service. So even though it's for the `rawDataDownloads` which is kind of related to meditrak, we would be best to avoid this. 

Could you check out how we map options to their labels with internal and/or DHIS2 data? That might reveal a more loosely coupled pattern